test/test-applications/integrations/Samples.HttpMessageHandler/RequestHelpers.cs

```diff
@@ -260,5 +260,24 @@ public static void SendHttpClientRequests(HttpClient client, bool tracingDisable
             }
         }
 #endif
+
+        public static async Task SendHttpMessageInvokerRequestsAsync(HttpMessageInvoker invoker, bool tracingDisabled, string url)
+        {
+            // Insert a call to the Tracer.Instance to include an AssemblyRef to Datadog.Trace assembly in the final executable
```

Comment:
Is this a ToDo, or just a note to make testing locally easier? 🙂

Reply:
Ups, It was a copy/paste thing from the previous method.

Reply:
Removed on 3485ca9